managed-services/transactional-email/reference-content/understanding-tem-reputation-score.mdx

```diff
@@ -56,7 +56,7 @@ Many factors might affect your domain's reputation. The most common are the foll
 
 - Emailing practices: The relevance, frequency, and timing of your emails are used to determine the validity of your domain.
 
-- Blacklists: Internet service providers and email providers may add domains and IPs to their blacklists to prevent spamming. Once your domain is blacklisted, your emails will be instantly flagged as spam. Your domain can also be blacklisted if a significant number of your recipients label your messages as spam.
+- Blacklists: internet service providers and email providers may add domains and IPs to their blacklists to prevent spamming. Once your domain is blacklisted, your emails will be instantly flagged as spam. Your domain can also be blacklisted if a significant number of your recipients label your messages as spam.
```

Comment:
```suggestion
- Blacklists: Internet service providers and email providers may add domains and IPs to their blacklists to prevent spamming. Once your domain is blacklisted, your emails will be instantly flagged as spam. Your domain can also be blacklisted if a significant number of your recipients label your messages as spam.
```